src/image/components/runtime.tsx

```diff
@@ -30,9 +41,33 @@ export const Runtime: React.FC<IProps> = ({ authoredState, report }) => {
     }
   };
 
+  const handleClick = () => {
+    const { url, highResUrl, credit, caption, creditLink } = authoredState;
+    const size = imageSize.current?.width && imageSize.current?.height
+                  ? { width: imageSize.current.width, height: imageSize.current.height }
+                  : undefined;
+    const allowUpscale = authoredState.scaling === "fitWidth";
+    const modalImageUrl = highResUrl || url;
+    const uuid = uuidv4();
+    const title = `<strong>${caption || ""}</strong> <em>${credit || ""}</em> ${ReactDOMServer.renderToString(getCreditLink() || <span/>)}`;
+    modalImageUrl && showModal({ uuid, type: "lightbox", url: modalImageUrl,
+                        isImage: true, size, allowUpscale, title });
+  }
+
+  const getCreditLink = () => {
+    const { creditLink, creditLinkDisplayText } = authoredState;
+    return creditLink && (
+      <div className={css.creditLink}>
+        <a href={creditLink} target="_blank">
```

Comment:
```suggestion
        <a href={creditLink} target="_blank" rel="noopener">
```
for [security reasons](https://web.dev/external-anchors-use-rel-noopener/).